doc/examples/viz_ui.py

```diff
@@ -123,8 +123,8 @@ def modify_button_callback(i_ren, obj, button):
 
 panel = ui.Panel2D(center=(440, 90), size=(300, 150), color=(1, 1, 1),
                    align="right")
-panel.add_element(button_example, 'relative', (0.2, 0.2))
-panel.add_element(second_button_example, 'absolute', (480, 100))
+panel.add_element(button_example, (0.2, 0.2))
+panel.add_element(second_button_example, (480, 100))
```

Comment:
Now, this button seems to be outside the panel, I think you should update the position if it is not on purpose.

Reply:
Done